src/stores/auth.ts

```diff
@@ -0,0 +1,37 @@
+import { defineStore } from 'pinia'
+import axios from 'axios'
+
+export const useAuthStore = defineStore({
+  id: 'auth',
+  state: () => ({
+    isLoggedIn: false,
```

Comment:
Why do we need an extra bool variable to verify whether the user is logged in or not?